assemble/conf/log4j-monitor.properties

```diff
@@ -16,23 +16,21 @@
 ## Log4j 1.2 file that configures logging for Accumulo Monitor
 ## The system properties referenced below are configured by accumulo-env.sh
 
-## Write out INFO and higher to log file
+## Define a log file appender
 log4j.appender.file=org.apache.log4j.RollingFileAppender
-log4j.appender.file.File=${accumulo.log.dir}/${accumulo.service.id}.log
+log4j.appender.file.File=${accumulo.log.dir}/${accumulo.application}.log
 log4j.appender.file.MaxFileSize=100MB
 log4j.appender.file.MaxBackupIndex=10
 log4j.appender.file.Threshold=INFO
 log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.ConversionPattern=%d{ISO8601} [%-8c{2}] %-5p: %m%n
 
-## Keep the last few log messages for display to the user in Monitor GUI
+## Define an appender for the Accumulo Monitor to log to its own web GUI
 log4j.appender.gui=org.apache.accumulo.server.monitor.LogService
-log4j.appender.gui.Keep=50
 log4j.appender.gui.Threshold=WARN
 
-## Log accumulo messages to file and gui appenders
-log4j.logger.org.apache.accumulo=INFO, file, gui
-log4j.additivity.org.apache.accumulo=false
+## Append monitor logs to its own web GUI
+log4j.logger.org.apache.accumulo=INHERITED, gui
```

Comment:
Yes, log4j documents "INHERITED" and "NULL" as synonyms for the default behavior of inheriting from the parent logger. It'd also be fine if we left it as "INFO", but I figured it'd make more sense to default to the user needing to adjust it in fewer places if they wished to change it.

INHERITED here means that it will inherit "INFO" from the root logger, but the `gui` appender is still going to filter out anything below "WARN".